sympy/external/tests/test_sage.py

```diff
@@ -222,6 +222,17 @@ def test_undefined_function():
     assert bool(sf(sx) == f(x)._sage_())
     #assert bool(f == sympy.sympify(sf))
 
+def test_abstract_function():
+    x,y = sympy.symbols('x y')
+    f = sympy.Function('f')
+    expr =  f(x,y)
+    sexpr = expr._sage_()
+    invexpr = sexpr._sympy_()
```

Comment:
Let's just leave this out for now. You can comment it out and leave a note to enable in the future.